azure-pipelines.yml

```diff
@@ -47,6 +47,7 @@ jobs:
       make ciprepare
       cd src/mainboard/emulation/qemu-q35
       make
+      make stack-sizes 2>/dev/null | grep '0x[0-9a-f]\+\s\+[0-9]\+\s\+_start'
```

Comment:
|| echo "Error: Didn't see stack size of _start. make stack-sizes might be broken."